mnq/sns-instances-notification-system/publisher-server/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM debian:bookworm-20240211
+LABEL authors="jlecomte@scaleway.com"
+
+RUN apt-get update && \
+    apt-get install -y golang ca-certificates && \
+    update-ca-certificates && \
+    rm -rf /var/lib/apt/lists/*
+
+WORKDIR /app
+
+COPY ./cmd ./cmd
+COPY ./internal ./internal
+COPY ./www ./www
+
+RUN go mod init publisher-server && \
```

Comment:
not mandatory but: just "go build" here and in the cmd just run the output file, because having "go run" as CMD will consume compile time at container start (same for other dockerfiles)